Volontulo/app/src/main/java/com/stxnext/volontulo/ui/volunteers/VolunteerDetailsAttendsFragment.java

```diff
@@ -0,0 +1,36 @@
+package com.stxnext.volontulo.ui.volunteers;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.stxnext.volontulo.R;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+public class VolunteerDetailsAttendsFragment extends Fragment {
```

Comment:
Why not used `VolontuloBaseFragment`? It has embedded ButterKnife functionality.